examples/3d/wireframe.rs

```diff
@@ -3,40 +3,37 @@
 use bevy::{
     pbr::wireframe::{Wireframe, WireframeConfig, WireframePlugin},
     prelude::*,
-    render::{render_resource::WgpuFeatures, settings::WgpuSettings, RenderPlugin},
+    render::{
+        render_resource::WgpuFeatures,
+        settings::{RenderCreation, WgpuSettings},
+        RenderPlugin,
+    },
 };
 
 fn main() {
     App::new()
         .add_plugins((
             DefaultPlugins.set(RenderPlugin {
-                render_creation: WgpuSettings {
+                render_creation: RenderCreation::Automatic(WgpuSettings {
+                    // WARN this is a native only feature. It will not work with webgl or webgpu
                     features: WgpuFeatures::POLYGON_MODE_LINE,
```

Comment:
Is this one of the features we don't enable by default? Because we always enable all wgpu features. Or did you just want to call this out specifically (which i agree we should)?

Reply:
We also need to mention this somewhere in the code itself (the module docs, plugin docs, material doc, or all 3 or something)

Reply:
I don't fully understand what exactly happens. I think there's some wgpu issue with how it works but essentially

- If I don't request the feature it works fine on native, but give an error saying that the feature is required but not requested when using webgl
- If I request the feature it works on native too, but now the error in webgl says that the feature is not supported.
- In both cases webgpu doesn't give an error but also doesn't render it correctly.

So I think this feature isn't being requested by default and it makes sense to request it but at the same time it doesn't seem to be necessary since it breaks either way.

Reply:
So this was indeed a wgpu bug and is now fixed https://github.com/gfx-rs/wgpu/pull/4196.

I think I'll keep the feature request because it is technically required for this feature and we don't request it by default.